demos/classification_demo/python/classification_demo.py

```diff
@@ -0,0 +1,308 @@
+                            default=0, type=int)
+    infer_args.add_argument('-nstreams', '--num_streams',
+                            help='Optional. Number of streams to use for inference on the CPU or/and GPU in throughput '
+                                 'mode (for HETERO and MULTI device cases use format '
+                                 '<device1>:<nstreams1>,<device2>:<nstreams2> or just <nstreams>).',
+                            default='', type=str)
+    infer_args.add_argument('-nthreads', '--num_threads', default=None, type=int,
+                            help='Optional. Number of threads to use for inference on CPU (including HETERO cases).')
+
+    io_args = parser.add_argument_group('Input/output options')
+    io_args.add_argument('--loop', default=False, action='store_true',
+                         help='Optional. Enable reading the input in a loop.')
+    io_args.add_argument('-o', '--output', required=False,
+                         help='Optional. Name of the output file(s) to save.')
+    io_args.add_argument('--pause', required=False, default=1, type=int,
+                         help='Optional. Pause in ms between frames to show.')
+    io_args.add_argument('-limit', '--output_limit', required=False, default=1000, type=int,
+                         help='Optional. Number of frames to store in output. '
+                              'If 0 is set, all frames are stored.')
+    io_args.add_argument('--no_show', help="Optional. Don't show output.", action='store_true')
+    io_args.add_argument('--output_resolution', default=None, type=resolution,
+                         help='Optional. Specify the maximum output window resolution '
+                              'in (width x height) format. Example: 1280x720. '
+                              'Input frame size used by default.')
+    io_args.add_argument('-u', '--utilization_monitors', default='', type=str,
+                         help='Optional. List of monitors to show initially.')
+
+    debug_args = parser.add_argument_group('Debug options')
+    debug_args.add_argument('-r', '--raw_output_message', help='Optional. Output inference results raw values showing.',
+                            default=False, action='store_true')
+    return parser
+
+
+def draw_labels(frame, classifications, output_transform, labels, gt_idx):
+    frame = output_transform.resize(frame)
+    class_id = classifications[0][0]
+    label_height = cv2.getTextSize(labels[class_id], cv2.FONT_HERSHEY_COMPLEX, 0.75, 2)[0][1]
+    initial_labels_pos =  frame.shape[0] - label_height * (2 * len(classifications) + 1)
+
+    if (initial_labels_pos < 0):
+        initial_labels_pos = label_height
+        log.warning('Too much labels to display on this frame, some will be omitted')
+    offset_y = initial_labels_pos
+
+    for cl in classifications:
+        class_id = cl[0]
+        conf = cl[1]
```

Comment:
```suggestion
    for class_id, conf in classifications:
```

Reply:
Done in 0c79d8f

Reply:
Apply the suggestion also to `print_raw_results` function, please

Reply:
thx! fixed in 6708930